test/integration_test_helper.rb

```diff
@@ -154,13 +154,13 @@ def fill_in_editor_field(css_locator, text)
     has_editor_display?(css_locator, text)
   end
 
-  def login_user(username, password)
+  def login_user(username, password, assert_current_path = true)
```

Comment:
We're not using this, please remove it :)
```suggestion
  def login_user(username, password)
```